lib/maze/hooks/logger_hooks.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Maze
+  module Hooks
+    # Hooks for Browser mode use
+    class LoggerHooks
+      class << self
+        def before(scenario)
+          location = "\"# #{scenario.location}\""
+          $logger.trace "\n--- Begin Scenario: #{scenario.name} #{location}"
```

Comment:
Looking at the resulting log file, the being/end aren't as clear as they could be imo.  Instead of a new line at the start here (and none in the end), how about an empty log line before this one and an empty one after the end.  Ie.:
```
$logger.trace ''
$logger.trace "\n--- Begin Scenario: #{scenario.name} #{location}"
```

and 

```
$logger.trace "--- End Scenario: #{scenario.name} #{location}"
$logger.trace ''
```

Just an idea to try and make separate scenarios stand out more.